Cargo.lock

```diff
@@ -443,6 +538,412 @@ dependencies = [
+ "cairo-lang-eq-solver",
+ "cairo-lang-sierra",
+ "cairo-lang-utils",
+ "itertools",
+ "thiserror",
+]
+
+[[package]]
+name = "cairo-lang-sierra-generator"
+version = "0.1.0"
+source = "git+https://github.com/eqlabs/cairo?branch=force-cairo-vm-0.1.0#fa91b1399ada5e09a5480731486e0b5567c6a03d"
+dependencies = [
+ "cairo-lang-debug",
+ "cairo-lang-defs",
+ "cairo-lang-diagnostics",
+ "cairo-lang-filesystem",
+ "cairo-lang-lowering",
+ "cairo-lang-parser",
+ "cairo-lang-plugins",
+ "cairo-lang-proc-macros",
+ "cairo-lang-semantic",
+ "cairo-lang-sierra",
+ "cairo-lang-syntax",
+ "cairo-lang-utils",
+ "id-arena",
+ "itertools",
+ "num-bigint 0.4.3",
+ "salsa",
+ "smol_str",
+]
+
+[[package]]
+name = "cairo-lang-sierra-to-casm"
+version = "0.1.0"
+source = "git+https://github.com/eqlabs/cairo?branch=force-cairo-vm-0.1.0#fa91b1399ada5e09a5480731486e0b5567c6a03d"
+dependencies = [
+ "assert_matches",
+ "cairo-lang-casm",
+ "cairo-lang-sierra",
+ "cairo-lang-sierra-ap-change",
+ "cairo-lang-sierra-gas",
+ "cairo-lang-utils",
+ "clap 4.1.1",
+ "itertools",
+ "log",
+ "num-bigint 0.4.3",
+ "num-traits 0.2.15",
+ "thiserror",
+]
+
+[[package]]
+name = "cairo-lang-starknet"
+version = "0.1.0"
+source = "git+https://github.com/eqlabs/cairo?branch=force-cairo-vm-0.1.0#fa91b1399ada5e09a5480731486e0b5567c6a03d"
+dependencies = [
+ "anyhow",
+ "cairo-lang-compiler",
+ "cairo-lang-defs",
+ "cairo-lang-diagnostics",
+ "cairo-lang-filesystem",
+ "cairo-lang-formatter",
+ "cairo-lang-lowering",
+ "cairo-lang-parser",
+ "cairo-lang-plugins",
+ "cairo-lang-semantic",
+ "cairo-lang-sierra",
+ "cairo-lang-sierra-ap-change",
+ "cairo-lang-sierra-gas",
+ "cairo-lang-sierra-generator",
+ "cairo-lang-sierra-to-casm",
+ "cairo-lang-syntax",
+ "cairo-lang-utils",
+ "clap 4.1.1",
+ "convert_case",
+ "genco",
+ "indoc",
+ "itertools",
+ "log",
+ "num-bigint 0.4.3",
+ "num-integer",
+ "num-traits 0.2.15",
+ "serde",
+ "serde_json",
+ "sha3",
+ "smol_str",
+ "thiserror",
+]
+
+[[package]]
+name = "cairo-lang-syntax"
+version = "0.1.0"
+source = "git+https://github.com/eqlabs/cairo?branch=force-cairo-vm-0.1.0#fa91b1399ada5e09a5480731486e0b5567c6a03d"
+dependencies = [
+ "cairo-lang-debug",
+ "cairo-lang-filesystem",
+ "cairo-lang-utils",
+ "salsa",
+ "smol_str",
+]
+
+[[package]]
+name = "cairo-lang-syntax-codegen"
+version = "0.1.0"
+source = "git+https://github.com/eqlabs/cairo?branch=force-cairo-vm-0.1.0#fa91b1399ada5e09a5480731486e0b5567c6a03d"
+dependencies = [
+ "cairo-lang-utils",
+ "genco",
+ "log",
+ "xshell",
+]
+
+[[package]]
+name = "cairo-lang-utils"
+version = "0.1.0"
+source = "git+https://github.com/eqlabs/cairo?branch=force-cairo-vm-0.1.0#fa91b1399ada5e09a5480731486e0b5567c6a03d"
+dependencies = [
+ "chrono",
+ "env_logger",
+ "indexmap",
+ "itertools",
+ "log",
+]
+
+[[package]]
+name = "cairo-vm"
+version = "0.1.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "0db448207a281d6ec29c4de24422fba5d52f0a0b9424dd7a758d995f5a89e40e"
```

Comment:
Wonder why it needs the vm..

Reply:
The dependency chain is as follows `cairo-vm`, `cairo-lang-casm`, `cairo-lang-sierra-to-casm`, `cairo-lang-starknet`